File: Stack.java
// Implementation of stack using array
public class Stack {
	public String[] stackArray;
	public int size;
	public int first = -1;
	
	public Stack(int size) {
		this.size = size;
		stackArray = new String[size];
	}
	
	public void push(String input) {
		if(first + 1 < size) {
			stackArray[++first] = input;
		} else {
			System.out.println("Stack is full");
		}
	}
	
	public String pop() {
		if(first >= 0) {
			String temp = stackArray[first];
			stackArray[first--] = null;
			return temp;
		} else {
			System.out.println("Stack is empty");
			return null;
		}
	}
	
	public String peek() {
		return stackArray[first];
	}

	public static void main(String[] args) {
		Stack stack = new Stack(2);
		stack.push("5");
		stack.push("2");
		System.out.println(stack.peek());
		stack.pop();
		System.out.println(stack.peek());
	}
}

File: Stack.py
# Implementation of stack using array
class Stack:
    def __init__(self, size):
        self.size = size
        self.first = -1
        self.stackArray = [None] * size

    def push(self, input):
        if self.first + 1 < self.size:
            self.first += 1
            self.stackArray[self.first] = input
        else:
            print("Stack is full")

    def pop(self):
        if self.first >= 0:
            temp = self.stackArray[self.first]
            self.stackArray[self.first] = None
            self.first -= 1
            return temp
        else:
			print("Stack is empty")
            return None

    def peek(self):
        return self.stackArray[self.first]

stack = Stack(2)
stack.push("5")
stack.push("2")
print(stack.peek())
stack.pop()
print(stack.peek())